lib/import/germline/genotype.rb

```diff
@@ -0,0 +1,505 @@
+                            'non-pathological variant' => 1,
+                            'benign' => 1 }.freeze
+
+      VARIANT_IMPACT_MAP = { 'missense' => 1,
+                             'nonsense' => 2,
+                             'frameshift' => 6 }.freeze
+
+      TESTING_TYPE_MAP = { 'askhenazi pre-screen' => nil,
+                           'diagnostic' => 1,
+                           'diagnostic test' => 1,
+                           'diagnostic testing' => 1,
+                           'mutation screening' => 1,
+                           'confirmation' => 1,
+                           'diag - symptoms' => 1,
+                           'diagnosis' => 1,
+                           'diagnosis (2)' => 1,
+                           'brca mainstreaming' => 1,
+                           'mainstreaming test' => 1,
+                           'predictive' => 2,
+                           'predictive testing' => 2,
+                           'presymptomatic test' => 2,
+                           'presymptomatic' => 2,
+                           'presymptomatic mlpa' => 2,
+                           'predictive test' => 2,
+                           'breast cancer predictives' => 2,
+                           'prenatal' => 3,
+                           'prenatal diagnosis' => 3 }.freeze
+
+      GENOME_BUILD_MAP = { 10 => 1,
+                           11 => 2,
+                           12 => 3,
+                           13 => 4,
+                           15 => 5,
+                           16 => 6,
+                           17 => 7,
+                           18 => 8,
+                           19 => 9,
+                           38 => 10 }.freeze
+
+      SPECIMEN_TYPE_MAP = {
+        'dna' => 12,
+        'blood' => 5,
+        'wax embedded' => 20,
+        'biopsy' => 20,
+        'mouthwash' => 10,
+        'saliva' => 10
+      }.merge((1..24).index_by(&:to_s)).freeze
+
+      BRCA_REGEX = /(?:b)?(?:r)?(?:c)?(?:a)?(1|2)/i
+
+      # ------------------------ Interogators ------------------------------
+      def positive?
+        cur_status = @attribute_map['teststatus'] # .map { |x| x == 6 }.
+        @logger.warn 'WARNING: reporting status, but none set' if cur_status.nil?
+        Maybe(cur_status == 2).or_else(false)
+      end
+
+      def full_screen?
+        scope = @attribute_map['genetictestscope']
+        return nil unless scope
+
+        scope == 'Full screen BRCA1 and BRCA2'
+      end
+
+      def reportidentifier
+        id = @attribute_map['servicereportidentifier']
+        log.warn 'Could not provide identifier' if id.nil?
+        Maybe(id).or_else('none_provided')
+      end
+
+      def get(key)
+        @attribute_map[key]
+      end
+
+      attr_reader :attribute_map, :pseudo_id1, :pseudo_id2, :raw_record
+
+      # ------------------------- Comparators ------------------------------
+      def similar_record(other_genotype, field_list, strict = true)
```

Comment:
Please ue kwargs when defining method with boolean argument - [example here](https://stackoverflow.com/questions/67391214/style-optionalbooleanparameter-use-keyword-arguments-when-defining-method-with)

Remember to change all calls of this method